test/staking/Integration.ts

```diff
@@ -456,26 +472,30 @@ describe("Integration", function () {
         mpondTokenId
       );
 
-      // console.log(pondRewards.map((a) => a.toString()));
-      // console.log(mpondRewards.map((a) => a.toString()));
+      console.log(pondRewards.map((a) => a.toString()));
+      console.log(mpondRewards.map((a) => a.toString()));
```

Comment:
comment out, don't print in the middle of tests

Reply:
commented